src/index.js

```diff
@@ -1,2 +1,25 @@
 import './style.css';
-import './modules/addScore.js';
+import * as API from './modules/LeaderboardAPI.js';
+
+const refresh = document.querySelector('.recent-scores-refresh-button');
+const form = document.querySelector('.add-your-scores-container');
+
+document.addEventListener('DOMContentLoaded', API.showScores);
+
+form.addEventListener('submit', (e) => {
+  console.log('are you working?');
+
+  e.preventDefault();
+  console.log('are you working?');
+  const input1 = document.querySelector('#name').value;
+  const input2 = document.querySelector('#score').value;
```

Comment:
[OPTIONAL] Well done, just remember to name your variables with meaningful names, this will make your code more professional and easier to read by other developers.

Reply:
@smunozmo thank you variables has been renamed for more meaning 
![e15](https://user-images.githubusercontent.com/40554384/162550889-a8912698-73f4-4813-b6c3-4412d1b94e56.PNG)

Reply:
Great! 💯